packages/disploy/src/structs/ChatInputInteraction.ts

```diff
@@ -9,7 +9,7 @@ export class ChatInputInteraction extends CommandInteraction {
 	 */
 	public options: ChatInputInteractionOptions;
 
-	public constructor(app: App, public raw: APIChatInputApplicationCommandInteraction) {
+	public constructor(app: App, public override raw: APIChatInputApplicationCommandInteraction) {
```

Comment:
While this would probably be a good idea to let the user access the raw objects; it is out of scope for this PR.